mappings/net/minecraft/item/SpawnEggItem.mapping

```diff
@@ -1,4 +1,20 @@
 CLASS ayx net/minecraft/item/SpawnEggItem
 	FIELD a SPAWN_EGGS Ljava/util/Map;
+	FIELD b primaryColor I
+	FIELD c secondaryColor I
+	FIELD d type Lahq;
+	METHOD <init> (Lahq;IILaxs$a;)V
+		ARG 1 type
+		ARG 2 primaryColor
+		ARG 3 secondaryColor
+		ARG 4 settings
+	METHOD a getColor (I)I
+		ARG 1 num
 	METHOD a forEntity (Lahq;)Layx;
+		ARG 0 type
+	METHOD a (Lhx;Lahq;)Z
+		ARG 1 tag
+		ARG 2 type
+	METHOD b deserializeEntityType (Lhx;)Lahq;
```

Comment:
use `to<DataStorageFormat>`/`from<DataStorageFormat>`-style lanuage, as per [the Contributing guideline draft](https://github.com/FabricMC/yarn/pull/477/files)

I feel like we might do a big issue on the `serialize/deserialize` thing changing things like `deserializeEntityType` to `entityTypeFromNBT` in the future.

Reply:
Changed in 5799ddbab63a45006e49404d1bf35c2485db65fc